scripts/docker-build.py

```diff
@@ -32,15 +32,39 @@ def build_image(
     if result.returncode != 0:
         raise Exception(f"Failed to build image with command {command_str}")
 
-
-def get_cuda_base_image(arch) -> str:
-    base_img = f"nvcr.io/nvidia/tensorrt:{TRT_CONTAINER_VERSION}-py3"
+def get_cuda_base_image(arch: str, container_version: str = CUDA_12_6_TRT_CONTAINER_VERSION) -> str:
+    base_img = f"nvcr.io/nvidia/tensorrt:{container_version}-py3"
     if arch == "arm64":
         # jetson has an integrated gpu
         base_img += "-igpu"
 
     return base_img
 
+def build_x86_specific_images(args):
```

Comment:
i reckon use consistent naming, minor point tho
```suggestion
def build_amd64_specific_images(args):
```